src/main/java/no/nav/rekrutteringsbistand/sms/rekrutteringsbistandsms/KandidatlisteClient.kt

```diff
@@ -0,0 +1,95 @@
+package no.nav.rekrutteringsbistand.sms.rekrutteringsbistandsms
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import no.nav.rekrutteringsbistand.sms.rekrutteringsbistandsms.utils.log
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.web.client.RestTemplateBuilder
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.MediaType.*
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter
+import org.springframework.stereotype.Service
+import org.springframework.util.MultiValueMapAdapter
+
+@Service
+class KandidatlisteClient(
+    @Value("\${azure.openid.config.token.endpoint}") private val tokenEndpoint: String,
+    @Value("\${azure.app.client.id}") private val clientId: String,
+    @Value("\${azure.app.client.secret}") private val clientSecret: String,
+    @Value("\${kandidat.api.scope}") private val kandidatapiScope: String,
+    restTemplateBuilder: RestTemplateBuilder,
```

Comment:
Vi har stort sett brukt fuel i steden for resttemplate senere tid sammen med kotli, burde vi ta en liten diskusjon på hva vi bruker fremover? Det går helt greit med resttemplate også.